test/message.integration.spec.ts

```diff
@@ -14,8 +14,12 @@ chai.use(chaiAsPromised);
 const { expect } = chai;
 
 const isWin = process.platform === 'win32';
-const isOSX = process.platform === 'darwin';
-const isCI = process.env['CI'] === 'true';
+const isLinux = process.platform === 'linux';
+const isDarwinArm64 = process.platform === 'darwin' && process.arch === 'arm64';
+const isLinuxArm64 = process.platform === 'linux' && process.arch === 'arm64';
+const isCirrusCi = process.env['CIRRUS_CI'] === 'true';
+const usesOctetStream =
```

Comment:
This makes me sad. Not your fault of course. We need to find a solution to this!

Reply:
I know, I'll card this up for a yak shave, now have we have a way of testing repeatably across platform, we can sort it, or document it